README.md

```diff
@@ -0,0 +1,83 @@
+# Team awesome
+
+Team awesome is a time entry application. An user can add time entries and team members to the application. The time entries are stored in the timesheets on the first page of the application. The time entries are displayed by date and gives the user the information about the client, work hours and duration per day.
+
+The Team Members page displays all the team members of the company, their role, client and starting date.
+
+## Technologies
+
+---
+
+- Node.js
+- Next.js
+- React and JSX
+- JSON
+- Vercel
+- Storybook
+
+## Installation
+
+---
+
+Install the lastest version of Homebrew
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Install Node Package Manager (NPM)
+
+```bash
+brew install node
+```
+
+After the installation of NPM initiate NPM:
+
+```bash
+npm init
+```
+
+Install Next.js, React and React-dom
+
+```bash
+npm install next react react-dom
+```
+
+Replace the `scripts` section of your `package.json` with the following:
+
+```bash
+"scripts": {
+  "dev": "next",
+  "build": "next build",
+  "start": "next start"
+}
+```
```

Comment:
I think this is already covered by your package.json and will be automatically installed by just npm i